frontend/packages/kubevirt-plugin/src/components/modals/menu-actions-modals/utils.ts

```diff
@@ -1,7 +1,6 @@
-import { VMGenericLikeEntityKind } from 'packages/kubevirt-plugin/src/types/vmLike';
-
 import { history } from '@console/internal/components/utils';
 import { getName, getNamespace } from '@console/shared/src/selectors/common';
+import { VMGenericLikeEntityKind } from 'packages/kubevirt-plugin/src/types/vmLike';
```

Comment:
Hmmm... wonder if this is valid syntax we should allow 🤔

Reply:
Well, technically its still a valid import but we want to follow the standard practices as much as possible. There might be a lot of instances like this across the repo but for the scope of this PR I didn't want to go into those changes.

Reply:
Understandable... I saw it and wanted to comment. Wasn't expecting you necessarily to solve this here; just sparking conversation.

You can mark this comment as resolved.